cscs-checks/system/slurm/slurm_check.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+# TODO: sinfo, check if the normal, long, debug, etc... partitions are present?
+# TODO: scontrol, do we want to scontrol something specific?
+@rfm.parameterized_test(['squeue'],
+                        ['sacct'],
+                        ['sinfo'],
+                        ['scontrol'])
+class SlurmCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self, variant):
```

Comment:
Use a relevant name instead of `variant`.

Reply:
Done